.ci/cico_updates_minikube.sh

```diff
@@ -98,12 +91,12 @@ testUpdates() {
 
   workspaceList=$(chectl workspace:list)
   workspaceID=$(echo "$workspaceList" | grep -oP '\bworkspace.*?\b')
+  echo $workspaceID
```

Comment:
Seems, If we really need this log line, then it should be more informative: `echo "[INFO] Workspace id started workspace is: ${workspaceID}"`. What do you think?

Reply:
+1